proto/zepben/protobuf/hc/LogMessage.proto

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+syntax = "proto3";
+option java_multiple_files = true;
+option java_package = "com.zepben.protobuf.hc";
+option csharp_namespace = "Zepben.Protobuf.HC";
+package zepben.protobuf.hc;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/struct.proto";
+
+message LogMessage {
+  /**
+   * The ID of the work package.
+   */
+  string workPackageId = 1;
+
+  /**
+   * The name of the scenario.
+   */
+  string scenario = 2;
+
+  /**
+   * The year in the scenario.
+   */
+  int32 year = 3;
+
+  /**
+   * The name of the feeder.
+   */
+  string feeder = 4;
+
+  /**
+   * Time step if this model was a single time step model. Null must be set if not.
+   */
+  oneof timeStep {
+    google.protobuf.NullValue timeStepNull = 5;
+    google.protobuf.Timestamp timeStepSet = 6;
+  }
+
+  /**
+   * The time at which this error occurred.
+   */
+  google.protobuf.Timestamp timestamp = 7;
+
+  /**
+   * The log level of this failure message.
+   */
+  LogLevel severity = 8;
+
+  /**
+   * The message to report.
+   */
+  string message = 9;
+
+}
```

Comment:
Question: should there be a source field just like in the logs table? This would make sense if failures come through in the same queue.

Reply:
yes - they do all currently get their own queue, but i don't want to rely on that.

Reply:
Added source field.